src/ppx_deriving_yojson_runtime.cppo.mli

```diff
@@ -1,4 +1,8 @@
+#if OCAML_VERSION >= (4, 08, 0)
+type 'a error_or = ('a, string) Result.t
+#else
```

Comment:
This should be unnecessary now that https://github.com/ocaml-ppx/ppx_deriving/pull/197 is merged.